locale/en-US/add-on.properties

```diff
@@ -203,7 +203,8 @@ door_hanger_button2=OK!
 infobar_screenshare_no_guest_message=As soon as your friend joins, they will be able to see any tab you click on.
 infobar_screenshare_browser_message2=You are sharing your tabs. Any tab you click on can be seen by your friends
 infobar_screenshare_browser_message3=You are now sharing your tabs. Your friend will see any tab you click on.
-infobar_screenshare_stop_sharing_message=You are no longer sharing your tabs
+infobar_screenshare_stop_sharing_message2=You are no longer sharing your tabs.
+infobar_screenshare_stop_no_guest_message=You have stopped sharing your tabs. When your friend joins, they won't be able to see anything until you restart sharing.
```

Comment:
This uses a regular single quote, rather than a unicode single quote. It came up in a recent thread that this will now turn tests orange on mozilla-central.  I've added a commit that fixes this.